### PR TITLE
Issue#7: add navigation bar

### PR DESCRIPTION
**Pull request by Anmavel**

### CODE REVIEW

File: frontend/src/component/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import "./Navigation.css"
+import {NavLink} from "react-router-dom";
+
+export default function Navigation() {
+
+    return (
+        <nav>
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+            <div className="navbar">
+                <NavLink to="/" className="active"><i className="fa fa-fw fa-home"></i> Home</NavLink>
```
Comment (valfink) [inline frontend/src/component/Navigation.tsx:11]: You can also install Font Awesome React packages and add the icons as components :) (it is a little bit complicated though)

File: frontend/src/component/Navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                <a href="#"><i className="fa fa-fw fa-user"></i> Login</a>
```
Comment (valfink) [inline frontend/src/component/Navigation.tsx:12]: This link doesn't have any functionality? There is no href and no onClick specified

File: frontend/src/page/SuppliesForm.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
+.form-submit submit-supply {
```
Comment (valfink) [inline frontend/src/page/SuppliesForm.css:1]: Did you maybe forget the dot here? (.submit-supply)

**Review by valfink — APPROVED** [review]
Good work 😊

File: frontend/src/component/Navigation.tsx (hunk 2 above)
Comment (Anmavel) [reply frontend/src/component/Navigation.tsx:12]: Tha link pick up the icons for the nav bar 👍

File: frontend/src/component/Navigation.tsx (hunk 1 above)
Comment (Anmavel) [reply frontend/src/component/Navigation.tsx:11]: i´ll take a look :-)

**Pull request merged**